web/src/main/java/com/bazaarvoice/emodb/web/EmoService.java

```diff
@@ -298,7 +301,7 @@ private void evaluateDatabus()
         ResourceRegistry resources = _injector.getInstance(ResourceRegistry.class);
         // Start the Databus service
         resources.addResource(_cluster, "emodb-bus-1",
-                new DatabusResource1(databus, databusClient, databusEventStore, databusResourcePoller));
+                new DatabusResource1(databus, databusClient, databusEventStore, databusEventWriterRegistry, databusResourcePoller, dataStore));
```

Comment:
I think we should avoid this injection of the writerRegistery and the datastore into the resource.

Reply:
see comment below